index.js

```diff
@@ -142,7 +142,7 @@ Nanologger.prototype._print = function (level) {
     args.push(obj)
   })
 
-  console.log.apply(console, args)
+  Function.prototype.apply.apply(console.log, [console, args])
```

Comment:
![](https://68.media.tumblr.com/b13df7fa70e61f42a048dd381a927d7e/tumblr_onj98svswF1rfd7lko1_500.gif)

Reply:
Maybe just add a comment.  Other Than that lgtm